src/components/Image/Image.spec.tsx

```diff
@@ -16,19 +16,14 @@ describe('Image', () => {
       'https://m.media-amazon.com/images/M/MV5BZGMwOGIwZjUtOWQ1OS00YWRjLWJmZGMtN2Y1OWQ3ZDYwYTM3XkEyXkFqcGdeQXVyNzU1NzE3NTg@._V1_.jpg',
     relative: 'c5f754_dd75514d14fa4057b4f4a6cc8ce7add3~mv2.jpg',
   };
+  const sampleDimensions = { width: 480, height: 360 };
```

Comment:
We need some dimensions (fixed/one+aspectRatio) to render the component properly